website/routes.py

```diff
@@ -337,6 +337,17 @@ def make_url_map(app):
             json_renderer,
         )
     ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/confirmed_emails/', 'get', auth_views.confirm_user_get, json_renderer),
+        Rule('/dashboard/confirmed_emails/', 'put', auth_views.add_confirmed_email, json_renderer)
+
+    ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/remove_confirmed_emails/', 'put', auth_views.confirm_email_remove, json_renderer)
```

Comment:
Use HTTP verbs rather than arbitrary naming conventions in URLs (see http://stackoverflow.com/a/2022938/1157536, points 1 and 2).

You should only need one URL to achieve the functionality you need:

```
GET /api/v1/emails/  -> list confirmed emails
POST /api/v1/emails/ -> add a new email
DELETE /api/v2/emails/ with JSON payload {"emails": "mick%40stones.com"} -> remove "mick@stones.com" from confirmed emails
```